encoding/rs/utils.go

```diff
@@ -14,7 +14,8 @@ import (
 // ToFrArray accept a byte array as an input, and converts it to an array of field elements
 //
 // TODO (litt3): it would be nice to rename this to "DeserializeFieldElements", as the counterpart to "SerializeFieldElements",
-//  but doing so would be a very large diff. I'm leaving this comment as a potential future cleanup.
+//
+//	but doing so would be a very large diff. I'm leaving this comment as a potential future cleanup.
```

Comment:
I really hate this formatting that gets automatically imposed! Must be a golang thing, since goland does the same thing :( No need to change, manually fixing this reformatting over and over is more trouble than it's worth

Reply:
really auto imposed here? It's odd it adds a line here. Could "TODO(litt3)" help? (no space after TODO)

Reply:
this happens when you leave a space at the beginning of a line. I think ` but` has a space in front of it, so it did this. Ill fix it

Reply:
[6feadaf](https://github.com/Layr-Labs/eigenda/pull/1104/commits/6feadaf01d5792ae1aac27568a7cefee3677767c)